TS Error with Color Tokens

## 🐛 Bug Report

The problem is [here](https://github.com/Workday/canvas-kit/blob/master/modules/react/tokens/index.ts#L25).
TLDR: The semantic colors has a colors key, and we’re exporting the default of colors as colors so it’s always being overwritten.

## To Reproduce

* Using TS ^3.9.9
* Look at `modules/react/tokens/index.ts`
* See the error

## Expected Behavior

Color tokens shouldn't be overwriting one another

## Actual Results

TS is angry.

![](https://media.giphy.com/media/11tTNkNy1SdXGg/giphy.gif)

## Browser (if applicable)

n/a

## Link to repl or repo (highly encouraged)

n/a

### Error Output

```bash
27   colors,
node_modules/@workday/canvas-kit-react/tokens/index.ts:27:3 - error TS2783: 'colors' is specified more than once, so this usage will be overwritten.
33   ...semanticColors,
node_modules/@workday/canvas-kit-react/tokens/index.ts:33:3
This spread always overwrites this property.
```

or

Include a screenshot of a visual bug

--- a/modules/react/tokens/index.ts
+++ b/modules/react/tokens/index.ts
@@ -1,4 +1,15 @@
-import * as canvasColorsWeb from '@workday/canvas-colors-web';
+import {
+  buttonColors,
+  chartingColorOffsets,
+  chartingColors,
+  colors,
+  commonColors,
+  gradients,
+  iconColors,
+  inputColors,
+  statusColors,
+  typeColors,
+} from '@workday/canvas-colors-web';
 
 import {borderRadius, CanvasBorderRadius, CanvasBorderRadiusKeys, CanvasBorderRadiusValues} from './lib/radius';
 import {BrandingColor, CanvasColor} from './lib/colors.types';
@@ -22,15 +33,21 @@ import {type,
 } from './lib/type';
 import {CSSProperties} from './lib/types';
 
-const {default: colors, gradients, ...semanticColors} = canvasColorsWeb;
 const canvas = {
+  buttonColors,
+  chartingColorOffsets,
+  chartingColors,
   colors,
+  commonColors,
   depth,
-  space,
-  type,
   fontFamily,
+  iconColors,
+  inputColors,
   monoFontFamily,
-  ...semanticColors,
+  space,
+  statusColors,
+  type,
+  typeColors,
 };
 
 export * from '@workday/canvas-colors-web';

--- a/modules/react/tokens/index.ts
+++ b/modules/react/tokens/index.ts
@@ -41,6 +41,7 @@ const canvas = {
   commonColors,
   depth,
   fontFamily,
+  gradients,
   iconColors,
   inputColors,
   monoFontFamily,

